jdaviz/configs/specviz/plugins/viewers.py

```diff
@@ -545,7 +545,22 @@ def _plot_uncertainties(self):
                 self.figure.marks = list(self.figure.marks) + [error_line_mark]
 
     def set_plot_axes(self):
-        # Set axes labels for the spectrum viewer
+        # Set y axes labels for the spectrum viewer
+        if y_unit.is_equivalent(u.Jy):
+            flux_unit_type = "Flux density"
+        elif y_unit.is_equivalent(u.Jy / u.sr):
+            flux_unit_type = "Surface brightness"
+        elif y_unit.is_equivalent(u.erg / (u.s * u.cm**2 * u.Hz)):
+            flux_unit_type = 'Flux'
+        elif y_unit.is_equivalent(u.e / u.s):
+            flux_unit_type = "Counts"
```

Comment:
"counts" is very ambiguous by STScI standard. I don't think this is correct.

Reply:
given that the unit is also in the label, I think this is probably ok?